src/js/pages/catalog/PackageDetailTab.js

```diff
@@ -310,8 +320,18 @@ class PackageDetailTab extends mixin(StoreMixin) {
                 </h1>
                 <p>{this.getPackageBadge(cosmosPackage, version)}</p>
               </div>
-              <div className="media-object-item">
+              <div className="media-object-item package-action-buttons">
                 {this.getInstallButtons(cosmosPackage)}
+                <p>
+                  By Deploying, you agree to {" "}
```

Comment:
@leemunroe shouldn't we lowercase "Deploying" - the current design in uppercased.